README.md

```diff
@@ -43,24 +43,40 @@ docker run -it --rm \
   truemark/aws-cli:latest
 ```
 
+Example using codeartifact support (OIDC + CODEARTIFACT)
+```bash
+docker run -it --rm \
+  -e AWS_OIDC_ROLE_ARN="YOUR_ACCESS_KEY_ID" \
+  -e AWS_WEB_IDENTITY_TOKEN="YOUR_OIDC_TOKEN" \
+  -e AWS_ASSUME_ROLE_ARN="YOUR_ROLE_ARN" \
+  -e AWS_ROLE_SESSION_NAME="YOUR_SESSION_NAME" \
+  -e AWS_CODEARTIFACT_TOOL="YOUR_CODEARTIFACT_TOOL" \
+  -e AWS_CODEARTIFACT_DOMAIN="YOUR_CODEARTIFACT_DOMAIN" \
+  -e AWS_CODEARTIFACT_REPO="YOUR_CODEARTIFACT_REPO" \
+  truemark/aws-cli:latest
+```
+
 # What are all the environment variables supported by this image?
 
 | Environment Variable        | Description                                                                             |
 |:----------------------------|:----------------------------------------------------------------------------------------|
- | AWS_ACCESS_KEY_ID           | Optional access key if using default AWS authentication.                                |
```

Comment:
Just wondering if we should align these.. OCD and all :)

Reply:
the unaligned rows are sorted and aligned in the new revision.